### ISSUE TITLE
lora 启用pissa 初始化进行sft 训练qwen-vl-4B 模型，保存模型时会卡死

### ISSUE THREAD

**Issue by verigle** [modelscope/ms-swift#9450]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new bug report. / 我已经搜索过现有的 issues，确认这是一个新的 bug report。

### Bug Description / Bug 描述

lora 微调模型 启用pissa 初始化进行sft 训练qwen-vl-4B 模型，保存模型时会卡死，每次是必现的

### How to Reproduce / 如何复现

lora 微调模型 启用pissa 初始化进行sft 训练

### Additional Information / 补充信息

保存模型那一步会卡死， 
/usr/local/lib/python3.12/site-packages/peft/peft_model.py:1308: UserWarning: PiSSA changes the base weights of the model and should thus not be used with other adapters. Consider converting the PiSSA adapter into a normal LoRA adapter: https://github.com/huggingface/peft/tree/main/examples/pissa_finetuning#convert-pissa-to-lora
  warnings.warn(msg)

<img width="2042" height="534" alt="Image" src="https://github.com/user-attachments/assets/09cbe214-ba60-4b3f-b274-945c3340fbe1" />

**Comment by Jintao-Huang** [comment]
有swift训练的脚本嘛

**Comment by verigle** [comment]
> 有swift训练的脚本嘛

swift sft \
    --model /data/modelzoo/Qwen/Qwen3-VL-4B-Instruct \
    --tuner_type lora \
    --lora_rank 64 \
    --lora_alpha 128 \
    --target_modules all-linear \
    --init_weights pissa \
    --torch_dtype bfloat16 \
    --dataset "$TRAIN_DATASET" \
    --val_dataset "$VAL_DATASET" \
    --external_plugins "$PLUGIN" \
    --eval_metric driver_status \
    --metric_for_best_model reward \
    --greater_is_better true \
    --num_train_epochs 3 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --learning_rate 2e-5 \
    --gradient_accumulation_steps 8 \
    --save_total_limit 5 \
    --logging_steps 10 \
    --eval_steps 50 \
    --save_strategy steps \
    --save_steps 1 \
    --output_dir "$OUTPUT_DIR" \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --max_length 4096 \
    --max_new_tokens 4096 \
    --generation_max_length 4096 \
    --max_pixels 524288 \
    --predict_with_generate true \
    --gradient_checkpointing true \
    --report_to tensorboard swanlab
